src/mdx/expandVersionedUrl.spec.js

```diff
@@ -23,7 +23,7 @@ describe("expandVersionedUrl", () => {
       ],
 
       [
-        "/Users/monkeypants/camunda-platform-docs/optimize_versioned_docs/version-3.9.0/what-is-optimize.md",
+        "/Users/monkeypants/camunda-platform-docs/optimize_versioned_docs/version-3.10.0/what-is-optimize.md",
```

Comment:
These tests depend on which versions are "current," so they get out of date with each release. It annoys me that they have to be updated over time, but (a) they don't run with CI so they don't block anything, and (b) it's not worth the effort to fix that, IMO. 

There are some changes in #2502 that will make it easier to mock the version mappings (moving them to a standalone file, which makes them mockable with Jest). Maybe in the future I will take advantage of that and make these tests more robust.